docker/compose.yml

```diff
@@ -20,6 +21,7 @@ services:
       - "127.0.0.1:21325:21326" # macos needs proxy to override the "origin" of the trezord request
     environment:
       - DISPLAY=docker.for.mac.host.internal:0
+      - PHYSICAL_TREZOR=$PHYSICAL_TREZOR
```

Comment:
Let's get rid of it here as it's a no go on mac.

Reply:
Done in force-push